File: demo_app/app/src/main/java/com/bandyer/demo_android_sdk/utils/networking/DeviceRegistrationInfo.java
/*
 * Copyright (C) 2019 Bandyer S.r.l. All Rights Reserved.
 * See LICENSE.txt for licensing information
 */

package com.bandyer.demo_android_sdk.utils.networking;

/**
 * @author kristiyan
 */
public class DeviceRegistrationInfo {
    String user_alias;
    String app_id;
    String push_token;
    String platform = "android";

    public DeviceRegistrationInfo(String user_alias, String app_id, String push_token) {
        this.user_alias = user_alias;
        this.app_id = app_id;
        this.push_token = push_token;
    }
}

File: app_utilities/src/main/java/com/bandyer/app_utilities/networking/DeviceRegistrationInfo.kt
/*
 * Copyright (C) 2020 Bandyer S.r.l. All Rights Reserved.
 * See LICENSE.txt for licensing information
 */
package com.bandyer.app_utilities.networking

/**
 * @author kristiyan
 */
class DeviceRegistrationInfo(var user_alias: String, var app_id: String, var push_token: String, var push_provider: String) {
    var platform = "android"

}